Mathlib/Computability/ContextFreeGrammar.lean

```diff
@@ -19,8 +19,11 @@ nonterminal symbol on the left-hand side of each rule.
 * `Language.IsContextFree.reverse`: The class of context-free languages is closed under reversal.
 -/
 
+open Function
```

Comment:
This is afaik used only in one section. Can we move it there please?

Reply:
It's used in two sections, actually

Reply:
My bad!